setup.py

```diff
@@ -291,7 +266,8 @@ def build_extension(self, ext):
     platforms=PLATFORMS,
     classifiers=CLASSIFIERS,
     package_dir=PACKAGE_DIR,
-    py_modules=PY_MODULES,
+    packages=[''],
+    package_data={'': ['__init__.py', 'brotli.py', 'brotli.pyi', 'py.typed']},
```

Comment:
if you don't fix this, then all the files get installed in the lib/pythonX.Y/site-packages folder directly without an enclosing package directory as it should be for valid python packages; even the empty `__init__.py` ends in there, or the `bro.py` script which we don't want to install at all.

Reply:
You're right.
I was only trying to keep the structure as it is. I will make the relevant changes later today.

Assuming a normal directory for the .py files,
then what you said is absolutely correct.

In that case bro.py could actually be shipped as ``__main__`` entry point of the module, so that it could be used as cli.
```python -m brotli {bro args}`` could then be used. It might not be that relevant for users, but since it's 0 effort and might be nice for tests.....

Reply:
> In that case bro.py could actually be shipped as __main__

sounds good to me.

Reply:
Nevermind, it works fine.